.github/workflows/ci.yml

```
@@ -14,6 +14,12 @@ jobs:
       run: yarn && yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript
     - name: Lint JS and LESS files
       run: npm run lint
+    - name: Assert LESS files formatting using Prettier
+      run: >
+        yarn add -D eslint-config-airbnb-base eslint-plugin-unicorn eslint-plugin-jest@^26.9.0 @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript github:fomantic/prettier#2.8.1-patched
```

The eslint packages have already been added by the linter before (and are only need for the linter anyway arent they?) So isnt 
```suggestion
        yarn add -D github:fomantic/prettier#2.8.1-patched
```
sufficient here?

---

And, i basically thought about having prettier a complete separate task (like lint or test or codeql).

---

Both eslint and stylelint are functional and formatting linters. Why should additional formatter be separated?

Later, once there is less patching and we do not require to support Node 12, I even prefer all these to be fixable with one command. So separate step does not make any sense to me.

---

ok, i just remember yourself saying "prettier is not a linter" 😉

---

> The eslint packages have already been added by the linter before (and are only needed for the linter anyway arent they?) So isnt
> 
> sufficient here?

What about this then?

---

of course, this was not intended, 👍